samples/streaming/consumer/src/main.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+mod streaming_consumer_config;
+
+use digital_twin_model::sdv_v1 as sdv;
+use env_logger::{Builder, Target};
+
+use image::io::Reader as ImageReader;
+use log::{info, LevelFilter};
+use samples_common::constants::{digital_twin_operation, digital_twin_protocol};
+use samples_common::utils::{
+    discover_digital_twin_provider_using_ibeji, retrieve_invehicle_digital_twin_uri,
+};
+use samples_protobuf_data_access::sample_grpc::v1::digital_twin_provider::StreamRequest;
+use samples_protobuf_data_access::sample_grpc::v1::digital_twin_provider::digital_twin_provider_client::DigitalTwinProviderClient;
+use show_image::{ImageView, ImageInfo, create_window, WindowProxy};
+use std::error::Error;
+use std::io::Cursor;
+use tokio_stream::StreamExt;
+use tonic::transport::Channel;
+use uuencode::uudecode;
+
+/// Perform the streaming.
+///
+/// # Arguments
+/// * `client` - The client connection to the service that will transfer the stream.
+/// * `num` - The number of images that we will stream.
```

Comment:
nit: this has been changed to number_of_images

Reply:
Fixed.